apps-rendering/src/components/metadata.tsx

```diff
@@ -237,6 +237,7 @@ const MetadataWithAlertSwitch: FC<Props> = ({ item }: Props) => {
 				{isLive(design) && (
 					<div css={css(toggleStyles, liveBlogPadding)}>
 						<ToggleSwitch
+							platform="ios"
```

Comment:
Is it worth adding this change to the PR description? Arguably, this is the _real_ change here and the bump to Source is what's needed to get here?

Reply:
Added 👍